source/sass/blocks/catalog/catalog-header.scss

```
@@ -0,0 +1,7 @@
+.catalog-header {
+  background-image: url("./img/back-mobile.png");
```

```suggestion
  background-image: url("../img/back-mobile.png");
```
вот так правильно, так как одна точка указывает на текущий каталог и браузер будет искать изображение в папке **css/img**, а такой папки нет. Двоеточие в относительном пути говорит о том, что нужно перейти на уровень вверх (относительно папки **css** это будет корень сайта) и уже там искать папку **img**.